doc/sample-data.md

```diff
@@ -0,0 +1,24 @@
+# Sample Data
```

Comment:
The name is surprising. maybe recorded sequences / files / camera data? or offline usage? 
we can also link to this doc from record and playback readme.